Gemfile.lock

```diff
@@ -186,16 +186,16 @@ GEM
     has_scope (0.7.2)
       actionpack (>= 4.1)
       activesupport (>= 4.1)
-    hashdiff (1.0.0)
+    hashdiff (1.0.1)
```

Comment:
### [hashdiff](https://github.com/liufengyun/hashdiff)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.0 → 1.0.1

[_[change-log](https://github.com/liufengyun/hashdiff/blob/master/changelog.md), [source-code](https://github.com/liufengyun/hashdiff)_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/liufengyun/hashdiff/compare/v1.0.0...v1.0.1).

These are the first 10 commits:
- (4edd29e) [Fix rubocop Style/FloatDivision violation](https://github.com/liufengyun/hashdiff/commit/4edd29e9b501f634de95f922da013caac3dfd7e6)
- (3e19fc9) [Make comparable? commutative](https://github.com/liufengyun/hashdiff/commit/3e19fc97029cae92f2bb17b52b4c1b9b95624b0c)
- (f8f0903) [Merge pull request #75 from dduugg/fix-rubocop-violation](https://github.com/liufengyun/hashdiff/commit/f8f0903e7c0a0052b192f7b6f333d23ca3b2d434)
- (b5011e2) [Merge pull request #76 from dduugg/symmetric-comparable](https://github.com/liufengyun/hashdiff/commit/b5011e2ec9e1d94d9caa223ec33d35d148d55faa)
- (1f4afd6) [fix invalid readme examples](https://github.com/liufengyun/hashdiff/commit/1f4afd67bd3896c704f671df665d13ea5d16cf6a)
- (513353c) [Merge pull request #78 from grosser/grosser/read](https://github.com/liufengyun/hashdiff/commit/513353cae9b0a31ec433410fecf6fe1e1aa279fb)
- (57f940b) [Added :indifferent option to ignore symbol vs string key dif…](https://github.com/liufengyun/hashdiff/commit/57f940b5173ed21cf9f59b80069bf266134b2551)
- (d36a075) [Work without ActiveSupport and add test case](https://github.com/liufengyun/hashdiff/commit/d36a0751888c364de518558d708fe2308c963f93)
- (ac8b2a7) [Add extra addition case in spec for :indifferent option](https://github.com/liufengyun/hashdiff/commit/ac8b2a7ed118f323753d02346485355d9767b8e0)
- (ee22ede) [Fix typo in README](https://github.com/liufengyun/hashdiff/commit/ee22ede18deb2eaf8e88788aca39a65bba44de80)


</details>